spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -237,6 +237,7 @@ class CodeEditor(TextEditBaseWidget):
     get_completions = Signal(bool)
     go_to_definition = Signal(str, int, int)
     sig_show_object_info = Signal(int)
+    sig_show_completion_object_info = Signal(str, str, bool)

```

Comment:
Could we cadd some comment regarding what the signature is?

# str: Title text, str: something text, bool: a check for whatever...
sig_show_completion_object_info = Signal(str, str, bool)

Makes it easier to read the signals